src/link/persisted-queries/index.ts

```diff
@@ -30,9 +30,12 @@ export interface ErrorResponse {
 type SHA256Function = (...args: any[]) => string | PromiseLike<string>;
 type GenerateHashFunction = (document: DocumentNode) => string | PromiseLike<string>;
 
+const PersistedQueryNotSupported = "PERSISTED_QUERY_NOT_SUPPORTED";
+const PersistedQueryNotFound = "PERSISTED_QUERY_NOT_FOUND"
```

Comment:
```suggestion
const PERSISTED_QUERY_NOT_SUPPORTED = "PERSISTED_QUERY_NOT_SUPPORTED";
const PERSISTED_QUERY_NOT_FOUND = "PERSISTED_QUERY_NOT_FOUND"
```

[nit] Since these are constants, could we name them with screaming snake case?